### ISSUE TITLE
React.ElementConfig is documented, but doesn't exist yet

### ISSUE THREAD

**Issue by Peeja** [facebook/flow#5592]
While I greatly appreciate the eager documentation, it seems the website is a bit overeager. [`React.ElementConfig<>` is already documented](https://flow.org/en/docs/react/hoc/#toc-supporting-defaultprops-with-react-elementconfig), but [the commit that adds it](https://github.com/facebook/flow/commit/91e85b8457671fe6c781873e37ae317e1402c830) hasn't been released yet. This led me to a whole lot of scrambling to try to understand why Flow couldn't find it.

Could the website deploy workflow be changed to only update with releases?

**Comment by lydell** [comment]
I’m really confused.

The 0.62.0 release (commit d273ca5b634253ba422eb657b9c5d787da1de34a) seems to have been done _after_ `React.ElementConfig` was added (commit 91e85b8457671fe6c781873e37ae317e1402c830). Still, I also experience that `React.ElementConfig` does not exist. Am I confused by the git commit log?

https://flow.org/try/#0PQKgBAAgZgNg9gdzCYAoVBLAtgBzgJwBdkwBDAZzACUBTUgY2KnzizACJ87H2BudKAFcAdowxxhYQjXKEAFAEoAXNW6EAdAFEYNLDWGEAwhKgYA5gB4AfGADeAX3RA

```js
/* @flow */

import * as React from "react";

function test(): React.ElementConfig<> {}
```

```
5: function test(): React.ElementConfig<> {}
                    ^ ElementConfig. Property not found in
3: import * as React from "react";
   ^ exports of `react`
```

**Comment by lydell** [comment]
Yay, it seems to be available in 0.63!

https://flow.org/try/#0PQKgBAAgZgNg9gdzCYAoVBLAtgBzgJwBdkwBDAZzACUBTUgY2KnzizACJ87H2BudKAFcAdowxxhYQjXKEAFKQBcYAPy0GhAHQBRGDSw1hhAMISoGAOYAeAOQATDADcbAPgCUy4YKwAjGvjAAb1QwMC5CQXxJAEZUAF90IA

**Comment by Peeja** [comment]
Huzzah!
